central/graphql/resolvers/vulnerabilities_v1.go

```diff
@@ -443,7 +447,21 @@ func (evr *EmbeddedVulnerabilityResolver) loadImages(ctx context.Context, query
 
 	query.Pagination = pagination
 
-	return evr.root.wrapImages(imageLoader.FromQuery(ctx, query))
+	if features.FlattenImageData.Enabled() {
+		resolvers, err := evr.root.wrapImageV2s(imageV2Loader.FromQuery(ctx, query))
+		res := make([]ImageResolver, 0, len(resolvers))
+		for _, resolver := range resolvers {
+			res = append(res, resolver)
+		}
+		return res, err
+	} else {
```

Comment:
no need for else, there is a return in the if

Reply:
For readability:
if feature flag enabled {
 // do stuff
 return
}
//do other stuff
return

is a pattern we've followed for readability